osu.Game/Storyboards/Drawables/DrawableStoryboardSample.cs

```diff
@@ -42,6 +42,14 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
             }
         }
 
+        protected override void SamplePlaybackDisabledChanged(ValueChangedEvent<bool> disabled)
```

Comment:
The fact that this doesn't call the `base` method means that if looping samples are currently (or ever are in the future) supported, they won't resume correctly. Is that intended behaviour?

Reply:
Yeah I forgot about the looping case earlier, you're right. Looping samples should be resumed (or rather restarted, currently) by this.